opencti-platform/opencti-graphql/config/schema/opencti.graphql

```diff
@@ -80,6 +80,18 @@ enum FilterMode {
   or
 }
 
+input Filter {
+  key: String
+  values: [String]
```

Comment:
yes, this will be changed
future schema for all the entity types:
![image](https://github.com/OpenCTI-Platform/opencti/assets/75783086/ee24d2b6-849f-403d-af31-c858bb5ed043)